Editor/Analytics/AmplitudeEditorLogger.cs

```diff
@@ -234,13 +233,12 @@ public void LogAvatarCreatorSampleImported()
             LogEvent(EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
         }
 
-        public void LogPackageInstalled(PackageCoreInfo packageInfo)
+        public void LogPackageInstalled(string id, string name)
         {
             LogEvent(EventName.INSTALL_PACKAGE, new Dictionary<string, object>
             {
-                { "id", packageInfo.Id },
-                { "name", packageInfo.Name },
-                { "url", packageInfo.Url },
```

Comment:
Nice spot to drop the url value which is never populated now. I was going to question it at first, but then remembered we now get the RPM package url version in the user properties ✅